src/applications/pact-act/constants/display-conditions/README.md

```diff
@@ -112,7 +112,7 @@ There are many different flows that would work for these display conditions. Her
 The **Short** flow (line(s) above the main flow line) includes follow-up questions regarding locations chosen and/or `Yes` responses that skip remaining questions in the batch (e.g. Burn Pit) and/or direct to a results page.
 
 The **Long** flow (main flow line) follows the `No` and `I'm not sure` response flows through all of the questions for that
-`SERVICE_PERIOD_SELECTION` leading to the results screens.
+`SERVICE_PERIOD_SELECTION` leading to the results pages.
```

Comment:
A few nitpicks for consistency in verbiage, and some more clarity about results pages mapping + how to get to results page 4.